website/docs/d/cloudant.html.markdown

```diff
@@ -0,0 +1,61 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_cloudant"
+description: |-
+  Get information about Cloudant instance.
+subcategory: "Cloudant"
+---
+
+# ibm_cloudant
+
+Provides a read-only data source for an existing IBM Cloud Cloudant service. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example Usage
+
+```hcl
+resource "ibm_resource_instance" "cloudant" {
+ 	  name     = "cloudant-service-name"
+ 	  service  = "cloudantnosqldb"
+ 	  plan     = "lite"
+ 	  location = "us-south"
+ }
+
+ data "ibm_cloudant" "instance" {
+ 	  name     = ibm_resource_instance.cloudant.name
+ }
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` (Required, String) The name of the IBM Cloudant resource instance.
+* `location` (Optional, String) The location or the environment in which instance exists.
+* `resource_group_id` (Optional, String) The id of the resource group in which the instance is present. If not provided it takes the default resource group.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `capacity` (Number) A number of blocks of throughput units. A block consists of 100 reads/sec, 50 writes/sec, and 5 global queries/sec of provisioned throughput capacity.
```

Comment:
I copied this description from here: https://github.com/cloudant/terraform-provider-ibm/pull/2/files#diff-cc8074261994ea5edaf50d0c0c8c912a77129f2caaa65f91a3e44c71f0fe2217R57
But I am okay to modify it here, just I think it would be also good to modify there, too.
Changed in 0eafc34f
WDYT about this new text?

Reply:
Thanks, the new text is fine. Yeah I didn't notice it when I was looking at the other PR, but I do think it is more important that the docs stay up to date in this particular respect than a code comment. One limitations is that AFAIK the Godoc comments are not markdown, so it'd have to be a little different anyway.